# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: js/src/jit-test/tests/basic/globals-shared-shapes.js
# Commit: 22854b710fbf
# Full Hash: 22854b710fbfb779ac6b19fe4c9ac2570189cec0
# Author: Jan de Mooij <jdemooij@mozilla.com>
# Date: 2021-03-06 09:33:16
# Regressor Bug: 1689413
# File Overlap Count: 1
# Description:
#   Bug 1689413 part 6 - Move Realm* from ObjectGroup to BaseShape. r=jonco
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D106976
# ==============================================================================

diff -r 4531d5c25694 -r 22854b710fbf js/src/jit-test/tests/basic/globals-shared-shapes.js
--- a/js/src/jit-test/tests/basic/globals-shared-shapes.js	Fri Mar 05 19:10:06 2021 +0000
+++ /dev/null	Thu Jan 01 00:00:00 1970 +0000
@@ -1,13 +0,0 @@
-// Test that objects in different compartments can have the same shape if they
-// have a null proto.
-
-var g1 = newGlobal();
-var g2 = newGlobal({sameZoneAs: g1});
-
-g1.eval("x1 = {foo: 1}");
-g2.eval("x2 = {foo: 2}");
-assertEq(unwrappedObjectsHaveSameShape(g1.x1, g2.x2), false);
-
-g1.eval("x1 = Object.create(null); x1.foo = 1;");
-g2.eval("x2 = Object.create(null); x2.foo = 2;");
-assertEq(unwrappedObjectsHaveSameShape(g1.x1, g2.x2), true);